src/main/java/com/shanebeestudios/skbee/elements/other/expressions/ExprBeaconEffect.java

```diff
@@ -0,0 +1,86 @@
+package com.shanebeestudios.skbee.elements.other.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Beacon;
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Beacon - Primary/Secondary Effect")
+@Description({"The primary/secondary potion effect of a beacon.",
+        "Note:",
```

Comment:
probably best to put this on another line
```suggestion
        "\nNote:",
```